libgalois/include/katana/PropertyGraph.h

```diff
@@ -45,8 +114,8 @@ struct KATANA_EXPORT GraphTopology {
 
   // TODO(amp): [[deprecated("use edges(node)")]]
   std::pair<Edge, Edge> edge_range(Node node_id) const {
-    auto edge_start = node_id > 0 ? out_indices->Value(node_id - 1) : 0;
-    auto edge_end = out_indices->Value(node_id);
+    auto edge_start = node_id > 0 ? adj_indices_[node_id - 1] : 0;
```

Comment:
Why don't we add `ASSERT` like `edge_dest()`?
`KATANA_LOG_DEBUG_ASSERT(node_id < adj_indices_.size());`

Reply:
done.